astropy/tests/command.py

```diff
@@ -1,8 +1,4 @@
-"""
-Implements the wrapper for the Astropy test runner in the form of the
-``./setup.py test`` distutils command.
-"""
-
+"""Implements the wrapper for the Astropy test runner."""
```

Comment:
Maybe add a note that this is just here for backward-compatibility for other packages and can be removed once astropy-helpers is no longer supported?